src/ansys/pyensight/core/dockerlauncher.py

```diff
@@ -159,6 +162,10 @@ def __init__(
             self._service_host_port["ws"] = self._get_host_port(
                 self._pim_instance.services["ws"].uri
             )
+            # if self._launch_webui:
```

Comment:
For the moment being I am not adding yet the webUI service in case of PyPim, since I need first to speak with the AnsysLab/PyPim team to get this optional service available; otherwise the code will always fail on AnsysLab